packages/libraries.python3.vm/tools/modules.xml

```diff
@@ -10,15 +11,15 @@
     <module name="msoffcrypto-tool"/>
     <module name="olefile"/>
     <module name="oletools"/>
+    <module name="pathlib"/>
```

Comment:
isn't this part of the standard library for Py3.9?

Reply:
you are right!